src/styles.scss

```diff
@@ -1241,7 +1245,8 @@ table th {
 }
 
 /* Don't have multi-line headers */
-.mat-header-row {
+.mat-header-row,
+.white-space-no-wrap {
```

Comment:
Does `.no-wrap` not work?

https://github.com/dockstore/dockstore-ui2/blob/5dbe9958aacb844b8fec7dd8501231722660c8d8/src/styles.scss#L987-L991

Reply:
It works but `text-overflow: ellipsis;`  and `overflow: hidden;` cause the link to be cut off:

![image](https://user-images.githubusercontent.com/97123241/222191787-38b4f01d-5a12-4cc0-b287-d431ef33949c.png)

Compared to just having `white-space: nowrap;`:

![image](https://user-images.githubusercontent.com/97123241/222193965-435f81ad-beac-4f2f-a31d-dd0282219415.png)